testnets/autheotestnet/assetlist.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "autheotestnet",
+  "assets": [
+    {
+      "description": "Native token of Autheo Chain",
+      "denom_units": [
+        {
+          "denom": "aauth",
+          "exponent": 0
+        },
+        {
+          "denom": "aauth",
+          "exponent": 5
+        }
+      ],
+      "base": "aauth",
+      "name": "Autheo Chain",
+      "display": "autheo",
+      "symbol": "THEO",
+      "logo_URIs": {
+        "png": "https://brown-changing-smelt-819.mypinata.cloud/ipfs/QmTNRXRnfr4z7bBYtZiSdVyQCBZkSwXzv9WPjnJJ8uQ5qe/logo.png"
```

Comment:
This png url must be pointing to this repo, and the PNG must be uploaded here, as well